lnd.go

```diff
@@ -124,9 +124,10 @@ func Main(lisCfg ListenerCfg) error {
 	}
 	cfg = loadedConfig
 	defer func() {
-		if logRotator != nil {
-			ltndLog.Info("Shutdown complete")
-			logRotator.Close()
+		ltndLog.Info("Shutdown complete")
+		err := logWriter.Close()
+		if err != nil {
+			fmt.Printf("Could not close log rotator: %v", err)
```

Comment:
is it possible to still use `logWriter.Errorf` if it failed to close? or is it assumed that it's already closed and that is the reason for `Printf`?

Reply:
it was my initial assumption that we probably couldn't use the logger any more after closing failed. but I looked into it and we should still be able to write to stout through the logger at least. so I fixed this.